### ISSUE TITLE
Document how to debug test-e2e locally

### ISSUE THREAD

**Issue by ValarDragon** [osmosis-labs/osmosis#1580]
## Background

We need to document how to debug test-e2e failures locally. 

## Suggested Design

The steps to debug test e2e as @nikever helpfully explained to me are:
* `make test-e2e`
* inspect the logs of the docker containers and see if something it’s there
* `docker ps -a`  # <--- list all docker containers
* note the container id of the one you want to see the logs
* and then docker logs <CONTAINER_ID>

Lets add this to contributing.md, and perhaps add a log line telling people to reference this for how to debug, on test-e2e failure?

## Acceptance Criteria

- [ ] Instructions in Contributing.md for how to debug `make test-e2e`
- [ ] (Maybe) link to instructions in the test-e2e failure message

**Comment by alexanderbez** [comment]
Yes, so this is why I originally added the `*_SKIP_CLEANUP` env var. By setting this, it keeps the docker infra intact and allows you to inspect containers, view logs, and even exec into them (if they're still running).

Are you looking for just a quick markdown doc here?

**Comment by ValarDragon** [comment]
yeah, just a quick inclusion of basic steps here in Contributing.md (or anywhere that makes sense)

Want to be able to easily link to instructions for how to fix it

**Comment by p0mvn** [comment]
There is also a README but it needs to be updated with the upgrade logic and the suggested debugging tips:

https://github.com/osmosis-labs/osmosis/blob/main/tests/e2e/README.md

**Comment by czarcas7ic** [comment]
Even better, we should print the logs out to the terminal when a docker container or test fails which would prevent us from needing to run it a second time locally.

But agreed, will work on more in depth instructions for this!

**Comment by alexanderbez** [comment]
Sounds like we can just update the README for now 👍
